h2o-extensions/steam/src/main/java/hex/steam/SteamHelloMessenger.java

```diff
@@ -23,12 +23,12 @@ public void onMessage(Map<String, String> message) {
         if (message.get(TYPE).equals("hello")) {
             assert sender != null : "Received message but sender is null";
             Map<String, String> response = new HashMap<>();
-            response.put(TYPE, "cloud_info");
+            response.put(TYPE, "hello_response");
```

Comment:
event type hello_response might be stored as a constant, because it is used multiple times

Reply:
in code its used only here, its better in tests to use the same string literal